vertical-pod-autoscaler/enhancements/xxxx-workload-admission/README.md

```diff
@@ -0,0 +1,61 @@
+# KEP-XXXX: Patch Well Known Controllers instead of Pods
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Design Details](#design-details)
+    - [Test Plan](#test-plan)
+- [Implementation History](#implementation-history)
+- [Alternatives](#alternatives)
+    - [Update the eviction API](#update-the-eviction-api)
+<!-- /toc -->
+
+## Summary
+
+The default behaviour of VPA Admission controller is to patch Pods when `UpdateModde`
+is different from `Off`. This works fine if we accept vertical scaling outside
+of standard deployments.
+
+For a wide range of application we cannot guarantee that such changes will not
+result in degraded performances, or worse, break the application. Because of that
+users might want to do that only as part of their regular deployment procedure. This
+makes even more sense when those deployments are done to multiple kubernetes clusters
+with safeguards between them.
+
+## Motivation
+
+The motivation behind the change is to give VPA users a way to apply resource
+changes only when they deploy their application.
+
+### Goals
+
+- Main: allow users to apply resource recommendation on deployment
```

Comment:
s/deployment/rollout

Should a `k rollout restart` apply the latest recommendation?

Reply:
changed, and yes `k rollout restart` should I think